internal/protos/orchestrator_service.pb.go

```diff
@@ -3,7 +3,7 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.31.0
+// 	protoc-gen-go v1.28.1
```

Comment:
Note protoc-gen-go was downgraded. Not sure if it matters

Reply:
Thanks for catching this. different versions of protoc-gen-go are used by developers to generate protobuf code, I think as long as tests run well, it should be fine. Will update my local protoc-gen-go version though.